### ISSUE TITLE
setting namedcell does not work correctly

### ISSUE THREAD

**Issue by GliderGeek** [vallettea/koala#114]
sample code:
```python
cwd = os.getcwd()
spreadsheet_path = os.path.join(cwd, 'template.xlsx')
graph = ExcelCompiler(spreadsheet_path).gen_graph()
graph.set_value('input1', 3)  # cell A1
result = graph.evaluate('Blad1!B1')  # =A1+5
# result=5 instead of 8
```

[template.xlsx](https://github.com/anthill/koala/files/1420037/template.xlsx)

**Comment by vallettea** [comment]
I think this is because you generated teh graph while A1 is empty and thus the graph has only one node. Try to assign a value in your worksheet before generating the graph.

**Comment by GliderGeek** [comment]
i don't think that is the case, because it does work when normal coordinates are used.
it would also be a lot nicer when the original graph can be re-used.
i think that i have found the problem in the code, will publish it here later

**Comment by Brad-eki** [comment]
I have been using named cells and they appear to be working properly.

has this been fixed?
